modules/trusted-firmware-m/Kconfig.tfm

```diff
@@ -294,8 +294,17 @@ config TFM_FLASH_MERGED_BINARY
 		(if building with TFM_BL2 is enabled), and the
 		Non-Secure application firmware.
 
+config TFM_LOG_LEVEL_SILENCE
+	bool "TF-M Disable secure output"
+	help
+	  Set all TF-M log levels to silence.
+	  On some platforms this could also reduce the code size by excluding
+	  the UART driver and the UART instance can be released from the secure
+	  image.
```

Comment:
Set the log level to silence for all TF-M modules (SPM, partition, etc.).
On some platforms this will release the UART from
the secure domain and reduce the uart driver's flash usage.